### ISSUE TITLE
Problem with compiling

### ISSUE THREAD

**Issue by BartlomiejBuczek** [failwyn/WebCompiler#69]
### Installed product versions
- Visual Studio: [17.4.4]
- This extension: [example 1.14.10]

### Current behavior
I have downloaded and instaled the extention although it doesn't show me the options to compile my files.

### Expected behavior
I should be able to compile my files
![image](https://user-images.githubusercontent.com/122546397/212103212-74638e92-cad0-447e-a636-49141b56efc0.png)
![image](https://user-images.githubusercontent.com/122546397/212103517-1ee28985-1027-4abf-82e8-a08bc70d6808.png)

**Comment by failwyn** [comment]
You need to have a project and you need to follow the Getting Started guide on the main page, the extension does not do anything without the compilerconfig.json file.

**Comment by BartlomiejBuczek** [comment]
k thx

**Comment by BartlomiejBuczek** [comment]
which project should I use for creating a front end/back end web page?

**Comment by failwyn** [comment]
Whichever one suits your project.

**Comment by BartlomiejBuczek** [comment]
![image](https://user-images.githubusercontent.com/122546397/212162860-82a04e94-d33f-4a2d-9c8c-3b181f3bb725.png)
![image](https://user-images.githubusercontent.com/122546397/212162909-05708474-c6fd-4b9d-9ea3-8921c279a210.png)
i still have a problem, in translation the error says: "the system cannot find the path specified" or something like this, what should I do now?

**Comment by failwyn** [comment]
Can you post your project so that I can replicate the issue?

**Comment by BartlomiejBuczek** [comment]
how do i post a project?

**Comment by BartlomiejBuczek** [comment]
basicaly that is why I asked which project would fit web design the best because it doesn't work on particular default projects.

**Comment by failwyn** [comment]
What project type does it not work with?

Zip the project, click the image button, choose the zip file.

**Comment by BartlomiejBuczek** [comment]
[WebApplication1.zip](https://github.com/failwyn/WebCompiler/files/10420167/WebApplication1.zip)
 Are you able to open this file? I zipped my project and it won't open on my computer now. Also the compiler file doesn't show on any of the projects at the moment.

**Comment by BartlomiejBuczek** [comment]
ok, but now i get this error which says that the system cannot find the specified path. What could be the problem? And should I have file nesting on or off? (it doesn't work with it nor without it)

**Comment by failwyn** [comment]
File nesting is a personal preference, it has no effect on the functionality of the extension. 

I need specific details to help; what file cannot be found? What did you change from the project you sent which is working as expected? I can only help if I can replicate your issue.

**Comment by BartlomiejBuczek** [comment]
the stylesheet1.scss file cannot be found

**Comment by failwyn** [comment]
What's throwing the file not found error? If WebCompiler cannot find an input file, it will display " Web Compiler found an error in compilerconfig.json". It doesn't display an error in the Error List.

It sounds like you are very new to development, if I were you, I would start with a basic website using html, css, and javascript, ,and get more accustomed to debugging in Visual Studio, then slowly start adding SCSS, TypeScript, MVC,  Razor, Angular, React, etc...  Identifying what Visual Studio is telling you in the Error list is going to be one of the most important skills you'll learn; debugging is 90% of development.

**Comment by BartlomiejBuczek** [comment]
yeah but my course is now teaching me about sass and I need to know how to deal with the basics. It is working on my friend's computer as well but there seems to be a problem with mine. I don't even know what this error means and I tried to search for it online but there is no explanation. I don't know if i should stick with manual compiler or should I hard reset my computer at this point.

**Comment by failwyn** [comment]
Try following these steps

[https://github.com/failwyn/WebCompiler/issues/63#issuecomment-1337660475](https://github.com/failwyn/WebCompiler/issues/63#issuecomment-1337660475)
